modules/candidate_list/templates/menu_candidate_list.tpl

```diff
@@ -44,6 +44,7 @@
                 </div>
             </div>
             <div class="row">
+                {if 1 < count($form.centerID.options)}
```

Comment:
this seems like an unnecessarily confusing way of writing it.. why not if count > 1?